octobot/src/server/admin.rs

```diff
@@ -344,7 +351,19 @@ impl MergeVersions {
             }
         }
 
-        self.make_resp(resp.set_versions(all_relevant_versions))
+        let version_url = if let Some(id) = all_relevant_versions.version_id {
+            Some(format!(
+                "{}/projects/{}/versions/{}",
+                resp.jira_base, &merge_req.project, id
+            ))
+        } else {
+            None
+        };
```

Comment:
played around with the `map` thing, since that definitely does feel more canonical. 

was able to get it to work by also modifying lines 308-309 above to `clone` before doing the `unwrap_or_default()`, since there was a partial move there, which is what prevented the borrow here. 

```
            jira_config.username = merge_req.admin_user.clone().unwrap_or_default();
            jira_config.password = merge_req.admin_pass.clone().unwrap_or_default();
```

with that change, this now works:
```suggestion
        let version_url = all_relevant_versions.version_id.map(|id| {
            format!(
                "{}/projects/{}/versions/{}",
                resp.jira_base, &merge_req.project, id
            )
        });
```

alternatively, it is a bit more idiomatic here to use `match` rather than an if/else block:
```suggestion
        let version_url = match all_relevant_versions.version_id {
            Some(id) => Some(format!(
                "{}/projects/{}/versions/{}",
                resp.jira_base, &merge_req.project, id
            )),
            None => None,
        };
```

Reply:
ty! the pattern match feels better than cloning the user/pass